managedwatchers.go

```diff
@@ -37,7 +37,6 @@ func (s *Service) AddProvisionWatcher(watcher contract.ProvisionWatcher) (id str
 	watcher.Origin = millis
 	watcher.Service = common.CurrentDeviceService
 	watcher.Profile = prf
-	common.LoggingClient.Debug(fmt.Sprintf("Adding Watcher: %s", watcher.Name))
```

Comment:
Why do you remove the debug logging here?
It should conform with other functions, for example
`common.LoggingClient.Debug(fmt.Sprintf("Removing managed ProvisionWatcher: %s", pw.Name))`

Just modify the sentence.

Reply:
I was thinking there's nearly identical debugging message [above](https://github.com/edgexfoundry/device-sdk-go/blob/7adc3885075d49ace02c324b1d4431dea73c6f6c/managedwatchers.go#L27),  should we still keep it ?